### DISCUSSION TITLE
Get statuses for a repo

### DISCUSSION THREAD

**Discussion by vromero** [community/community#24904] [Apps, API and Webhooks]
<p>I’d like to get all the statuses context and possibly latest target_url for a repository, in a similar fashion to what the UI for branch protection do show:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/5/562549524d004fd8d894696fa6b20564ef01c3d9.png" data-download-href="/uploads/short-url/ci4WMZhXdAe5jRyLTExHXTr3lax.png?dl=1" title="ZGzNd.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/1119553/181087424-46820295-95d4-4a80-838f-4b810ed7c188.png" alt="ZGzNd.png" data-base62-sha1="ci4WMZhXdAe5jRyLTExHXTr3lax" width="690" height="331" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/5/562549524d004fd8d894696fa6b20564ef01c3d9_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">ZGzNd.png</span><span class="informations">707×340 32.6 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Can I ask if anyone is aware of a single API request that can perform such task or otherwise a reduced combination of requests. I know it can be done in a rather brutal way, but I’m hoping for something light on github requests.</p>
<p>Thanks</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>Unfortunately, there isn’t an API endpoint for what you’re describing. I’ll take the suggestion back to the API team, however I can’t promise anything or give an ETA for when it might happen.</p>
<p>Thanks!</p>
